app/src/main/java/com/example/marvelapp/presentation/fragment/MarvelDetailsFragment.kt

```diff
@@ -0,0 +1,36 @@
+package com.example.marvelapp.presentation.fragment
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.DialogFragment
+import com.example.marvelapp.databinding.CharacterDetailLayoutBinding
+
+class MarvelDetailsFragment : DialogFragment() {
+    private lateinit var binding: CharacterDetailLayoutBinding
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        binding = CharacterDetailLayoutBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
```

Comment:
If this method just make super.onViewCreated(view, savedInstanceState) should be removed.

Reply:
Done thx